default/src/components/asaTransfer.js

```diff
@@ -0,0 +1,69 @@
+/* global AlgoSigner */
+import algosdk from 'algosdk';
+
+import { networks } from '../../algob.config';
```

Comment:
just for note: we don't want to do this here (importing config from `algob.config.js`). So after algob integration this should be removed.